firmware/quintuna/RSM/src/io/io_suspension.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <stdbool.h>
+
+#ifdef TARGET_EMBEDDED
+#include "hw_adc.h"
+
+typedef struct
+{
+    const AdcChannel *src;
+} Suspension;
+#else
+#include "app_utils.h"
+EMPTY_STRUCT(Suspension);
+#endif
+
+float io_suspension_getTravel(const Suspension *suspension);
```

Comment:
i think in this case, it is probably better to get rid of the struct and just have two functions, one for each suspension travel